<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAddressBookBase</class>
 <widget class="QWidget" name="DlgAddressBookBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Addressbook settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Collections</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTreeView" name="m_pItemBackendW">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="m_pEditCollection">
           <property name="text">
            <string>Edit collection</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pAddCollection">
           <property name="text">
            <string>Add collection</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Contact Filter</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="kcfg_hideContactWithoutPhone">
         <property name="text">
          <string>Hide contacts without phone numbers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_pPhonetypeL">
         <property name="text">
          <string>Use contact from the following phone number category:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="m_pPhoneTypeList">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Visible Contact Details</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="kcfg_displayOrganisation">
         <property name="text">
          <string>Display organisation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_displayEmail">
         <property name="text">
          <string>Display email address</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pAddCollection</sender>
   <signal>clicked()</signal>
   <receiver>DlgAddressBookBase</receiver>
   <slot>slotAddCollection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>661</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pEditCollection</sender>
   <signal>clicked()</signal>
   <receiver>DlgAddressBookBase</receiver>
   <slot>slotEditCollection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>661</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotEditCollection()</slot>
  <slot>slotAddCollection()</slot>
 </slots>
</ui>
